### PR TITLE
tp-qemu: Add README to cpuid test, and fix cfg file issue

### PR DESCRIPTION
**Pull request by suqinhuang**
Add README to cpuid test, update the cfg format to make it the same with other cfg files and fix config issue

### CODE REVIEW

**Comment by xutian** [comment]
Can one of the admins verify this patch?

**Comment by XueqiangWei** [comment]
LGTM,
Acked by Xueqiang Wei xuwei@redhat.com

**Comment by xutian** [comment]
LGTM,

Acked-by: Xu Tian <xutian@redhat.com>

**Comment by xutian** [comment]
@suqinhuang 

ID please .

**Comment by suqinhuang** [comment]
ID: 1168562

File: qemu/tests/cfg/qemu_cpu.cfg [hunk 1]
```diff
@@ -18,19 +18,26 @@
...
                         - 6_0:
                         - 6_1:
                         - 6_2:
                         - 6_3:
                         - 6_4:
-        - upstream:
-            variants qemu_version:
+                - 7:
+                    only Host_RHEL.7
+                    variants:
```
Comment (xutian) [inline qemu/tests/cfg/qemu_cpu.cfg]: see alots of such variants but not see where used it?  so any difference about 7_0 and 7_1 ? and any differences between  variants 6 and 7 ?

Comment (suqinhuang) [inline qemu/tests/cfg/qemu_cpu.cfg]: some scenerial tested on Host_RHEL.6 only or Host_RHEL.7 only, 
no diff between 6.xx , or 7.xx until you set it. didn't remove 6.xx or  as it's set by the owner, I just just update the format

Comment (suqinhuang) [inline qemu/tests/cfg/qemu_cpu.cfg]: some new cpu model support on rhel7 only, 
some old cpu model don't support on rhel6 and rhel7

 cpu_model_intel.486, cpu_model_intel.pentium, cpu_model_intel.pentium2, cpu_model_intel.pentium3:
no qemu_flavor_rhel.6..host_cpu_vendor_intel
no qemu_flavor_unknown..host_cpu_vendor_intel
no qemu_flavor_rhel.6..host_cpu_vendor_unknown

**Comment by xutian** [comment]
Acked-by: Xu Tian <xutian@redhat.com>

**Comment by xutian** [comment]
Merged. thanks @suqinhuang  and @XueqiangWei

**Pull request merged**